commands/get.js

```diff
@@ -301,6 +356,11 @@ export function builder (yargs) {
       type: 'boolean',
       default: false
     })
+    .option('max-inflight-requests', {
+      describe: 'How many download resource can be downloaded at the same time',
+      type: 'number',
+      default: 10
```

Comment:
The name of the option is hard to explain. Please align up- and download options.